js/vod-stream-embed.js

```diff
@@ -0,0 +1,85 @@
+// Video-on-demand Stream Video
+
+var ipfs_gateway_self = 'https://ipfs.io'; // IPFS gateway of this node
+var ipfs_gateway_origin = 'https://ipfs.io'; // IPFS gateway of origin stream
+var m3u8_ipfs = 'https://ipfs.io/ipfs/QmZJ5ZbJTfhQXM57PSWYy1VsHRDfL5AckjXy6W7RmP89AX'; // File path to m3u8 with IPFS content via HTTP server
```

Comment:
This is hard coded to Opening Talk `QmZJ5ZbJTfhQXM57PSWYy1VsHRDfL5AckjXy6W7RmP89AX` here, but I imagine we can make a list with all pinned hashes, and have the list below the player change this property. 

@garrying @dcwalk does that sound reasonable to do?